### ISSUE TITLE
Target group configuration could include better validation

### ISSUE THREAD

**Issue by markusl** [aws/aws-cdk#3703]
### General Information
If user accidentally writes the following code, it will be rejected when deploying:
```javascript
target.configureHealthCheck({
        interval: cdk.Duration.seconds(60),
        timeout: cdk.Duration.seconds(60),
        healthyThresholdCount: 2,
        unhealthyThresholdCount: 5,
      })
```

`Health check interval must be greater than the timeout. (Service: AmazonElasticLoadBalancingV2; Status Code: 400; Error Code: ValidationError; Request`

### Proposed Solution
This kind of validations could be done already in CDK to speed up the development cycle.

### Environment

  - **CDK CLI Version: 1.4.0**

**Comment by bl-ue** [comment]
Shouldn't this be closed now that #16107 has been merged?

**Comment by markusl** [comment]
Thanks for noticing. I'll assume this has been implemented :)
